splunk_connect_for_snmp_poller/manager/task_utilities.py

```diff
@@ -778,3 +779,14 @@ def build_contextData(version, community, server_config):
         return ContextData(contextEngineId, contextName)
     except Exception as e:
         logger.error(f"Error happend while building ContextData: {e}")
+
+
+def is_oid(profile: str) -> bool:
+    """
+    This function checks if profile is an OID. OID is defined as a string of format:
+        - ex. 1.3.6.1.2.1.2
+        - ex. 1.3.6.2.1.*
+    @param profile: variable from inventory file, can be a name of profile or an OID
+    @return: if the profile is of OID structure
+    """
+    return bool(re.match(r"^(\d+\.?)*\*?$", profile))
```

Comment:
This pattern would match "232." and empty string. I've found sth more detailed in the net:
^([1-9][0-9]{0,3}|0)(\.([1-9][0-9]{0,3}|0)){5,13}$

Reply:
As discussed, we don't need something as detailed to distinguish OID from profile